saas/static/js/djaodjin-saas-vue.js

```diff
@@ -411,6 +411,16 @@ var regions = {
 
 var DATE_FORMAT = 'MMM DD, YYYY';
 
+function handleRequestError(res){
```

Comment:
I am pretty sure `showErrorMessages` takes care of all cases which have been `if/then` here.

Reply:
It wasn't working, that's why I wrote this function. Should I make `showErrorMessages` work? Besides you said that we are moving away from jQuery dependency which would require us to use an AJAX library like axios, at which point we can also refactor the error handling.

Reply:
BTW, by not working I mean that `showErrorMessages` would dump a JS stack trace (which is not very user-friendly), so maybe it expects an Angular AJAX format response instead of jQuery.